resources/CSS/style.css

```diff
@@ -249,3 +250,87 @@ ul {
 .bor-bot {
   border-bottom: 1.5px solid #dfe1e6;
 }
+
+/* Footer */
+
+footer {
+  background-color: #fff;
+}
+
+.footer-content {
+  position: relative;
+  background-color: #6070ff;
+  border-top-left-radius: 100px;
+  padding: 60px 30px 45px 30px;
+}
+
+.footer-content h2 {
+  color: #fff;
+  text-align: center;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.footer-content p {
+  text-align: center;
+  color: #fff;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.footer-content img {
+  position: absolute;
+  right: 0;
+  top: 10px;
+  width: 220px;
+  height: 709px;
+}
+
+form {
+  margin-top: 50px;
+}
+
```

Comment:
- [ ] As required for the project, kindly make sure you use `Flexbox` to layout the `footer-content` or `footer elements` and your project will be completed successfully